addon/modifiers/Modifiers.stories.mdx

```diff
@@ -11,19 +11,19 @@ import { Meta } from '@storybook/addon-docs/blocks';
   </blockquote>
   <figcaption>— Ember JS</figcaption>
 </figure>
 
-| Modifier         | Description                                                                  | Usage                                                                          | Default                                                                                                   |
-|------------------|------------------------------------------------------------------------------|--------------------------------------------------------------------------------| ----------------------------------------------------------------------------------------------------------|
-| enable-popover   | Adds a Bootstrap's Popover to the element                                    | `<button {{enable-popover [trigger]}}>Popover</button>`                        | Default Trigger : **hover**                                                                               |
-| enable-dropdown  | Toggles Bootstrap's dropdown on an element                                   | `<a {{enable-dropdown}}>Dropdown Trigger link/button</a>`                      |                                                                                                           |
-| on-click-outside | Triggers the passed callback function when a user clicks outside the element | `<div {{on-click-outside callbackFunction}}>Floating Div</div>`                |                                                                                                           |
-| on-bottom-reached | Triggers the passed callback function when a user reaches the bottom of the target element | `<div {{on-bottom-reached callbackFunction}}>Div</div>`                |                                                                                                           |
-| scroll-shadow    | Add bottom shadow for scrollable element                                     | `<div {{scroll-shadow color="field"}}>`                                        | color: "default" (white)                                                                                           |
+| Modifier               | Description                                                                                | Usage                                                           | Default                     |
+| ---------------------- | ------------------------------------------------------------------------------------------ | --------------------------------------------------------------- | --------------------------- |
+| enable-popover         | Adds a Bootstrap's Popover to the element                                                  | `<button {{enable-popover [trigger]}}>Popover</button>`         | Default Trigger : **hover** |
+| enable-input-autofocus | Focuses on selected input with smooth scroll                                               | `<input type="text" {{enable-input-autofocus}}/>`               |                             |
```

Comment:
the* selected input 
but also, that's not completely true because you've handled the case where we can add the modifier on a parent element and the modifier will still target the first available input;